test/spec/token.js

```diff
@@ -1099,6 +1835,32 @@ define(function(require) {
       }
     });
 
+    util.itMakesCorrectRequestResponse({
```

Comment:
we don't; they're on my list! They'll be tested when we break out the getOAuthUrls method